### DISCUSSION TITLE
Error in Update-Databse

### DISCUSSION THREAD

**Discussion by asgharb** [jasontaylordev/CleanArchitecture#584] [General]
hi.
When I try to update the database, I get the following error:

Unable to resolve service for type 'Microsoft.EntityFrameworkCore.Migrations.IMigrator'. This is often because no database provider has been configured for this DbContext. A provider can be configured by overriding the 'DbContext.OnConfiguring' method or by using 'AddDbContext' on the application service provider. If 'AddDbContext' is used, then also ensure that your DbContext type accepts a DbContextOptions<TContext> object in its constructor and passes it to the base constructor for DbContext.

**Comment by deceptiveSimplicity** [comment]
I think this is because u are using InMemory-DB. For upgrading a DB u must use a real DB-provider. Just check your Configuration if there is "UseInMemoryDatabase" set to true.
